### DISCUSSION TITLE
Nested Routing does not seem to work for me.

### DISCUSSION THREAD

**Discussion by dikkadev** [leptos-rs/leptos#1224] [Q&A] [answered]
To improve how my site works (sidebar + workspace area) I want to use context in the URL and nested routing to load the correct data for the workspace. I’ve tried around a bit without anything working, then tried to use the example from the book.

```rust
<Router>
    <main class="text-slate-100 bg-black flex flex-row h-screen w-screen select-anaheim">
        <Routes>
            <Route path="/contacts" view=|cx| view! { cx, "list" }>
                <Route path=":id" view=|cx| view! { cx, "info" }/>
                <Route path="" view=|cx| view! { cx, "nothing" }/>
            </Route>
            //... routes for login and create
        </Routes>
    </main>
</Router>
```

Behavior:
| route            | expected       | actual |   |
|------------------|----------------|--------|---|
| `/contacts`      | `list`         | `list` | ✓ |
| `/contacts/cool` | `list cool`    | `list` | ✗ |
| `/contacts/`     | `list nothing` | 404    | ✗ |

I am currently running with commit [1d7235d](https://github.com/leptos-rs/leptos/commit/1d7235d4ca4b8f8cabdf0fa66f823b0b742e422a). I get the same behavior with `0.3.1`

**Comment by gbj** [comment, marked as answer]
Your parent route doesn't have an `<Outlet/>` component so it doesn't show the child anywhere.

**Reply by dikkadev** [reply]
One would think over time you learn to read stuff twice or more before asking questions, instead of skimming over it and thinking you got it :). Somehow I thought that an `<Outlet/>` was optional, and the children would be put at the end if it wasn't there.
